calculator.html

```diff
@@ -183,40 +183,45 @@
         numCheck  = 0
       }
     }
-      
+
     function equals(){
       var val = parseFloat(digit.join(''))
-      var value   = val
-      if(signCheck==4) {
-        var result = storage * value
-        console.log(storage + " x " + value + " = " + result)           
-        storage    = result
-        console.log ("the stored value is now " + storage)
-      }
-      if(signCheck==3) {
-        if(value!=0){
-        var result = storage / value
-        console.log(storage + " / " + value + " = " + result)          
-        storage    = result
-        console.log ("the stored value is now " + storage)
-        }
-        else{
-          console.log("can't divide by 0!")
-        }
-      }
-      if(signCheck==2) {
-        var result = storage - value
-        console.log(storage + " - " + value + " = " + result)
-        storage = result
-        console.log ("the stored value is now " + storage)
-      }
-      if(signCheck==1) {
-        var result = storage + value
-        console.log(storage + " + " + value + " = " + result)
-        storage    = result
-        console.log ("the stored value is now " + storage)
+      var value = val
+      var result = null;
```

Comment:
if we define result up here first, we know if our our calculation logic below will never populate it it.